### ISSUE TITLE
Use existing crossing nodes when making segments

### ISSUE THREAD

**Issue by 3vivekb** [AccessMap/crossify#5]
If existing crossing nodes exist in a segment, use them.  

It's an open question whether to use the sidewalk nodes to angle towards that crossing or to use that crossing node and generate two lines that reach the existing sidewalks in the shortest distance.

**Comment by nbolten** [comment]
Do you mean using `highway=crossing` nodes from the street data as a hint for crossing placement?

**Comment by 3vivekb** [comment]
Yes.  Exactly.  If they exist at the intersection.  But mostly the crossing would go through it then in JOSM we could join the two with a single click.

**Comment by nbolten** [comment]
Ah, I understand. Requiring that the path go through the existing node is a very different strategy from what `crossify` currently supports, so it will probably be a bit of time before this feature is implemented.

**Comment by 3vivekb** [comment]
Understood.
